data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -449,10 +473,39 @@
             id="Al'Tar"
         [/filter]
 
-        [message]
-            speaker=second_unit
-            message= _ "Not a very bright one, was he?"
-        [/message]
+        [switch]
+            variable=second_unit.race
+            [case]
+                value="human"
+                [message]
+                    speaker=second_unit
+                    message= _ "Not a very bright one, was he?"
+                [/message]
+            [/case]
+            [case]
+                value="orc"
+                [if]
+                    [have_unit]
+                        side=3
+                        id=Garrugch
+                    [/have_unit]
+                    [then]
+                        [message]
+                            speaker=second_unit
+                            #po: "him" is referring to Al'Tar
+                            message= _ "We got him, chief!"
+                        [/message]
+                    [/then]
+                    [else]
+                        [message]
+                            speaker=second_unit
+                            message= _ "We avenged our chief..."
```

Comment:
Garrugch is not a chief, though. So you should probably switch this to leader or something